src/app/services/auth.service.ts

```diff
@@ -34,8 +38,13 @@ export class AuthService {
 
   private manager: UserManager = new UserManager(getClientSettings());
   private user: User | null;
+  private Backenduser: BackendUser | null;
+
+  protected http: HttpClient;
+
+  constructor(http: HttpClient) {
+    this.http = http;
 
-  constructor() {
     this.manager.getUser().then((user) => {
```

Comment:
Logic should not be excuted in the constructor but in the ngOnInit. See the following article for more background info: https://indepth.dev/the-essential-difference-between-constructor-and-ngoninit-in-angular/

Reply:
the logic was already there, i only added the httpclient.

Reply:
Could you still move it to the ngOnInit?

Reply:
Nice, I didn't know this :)